semcore/popper/src/style/popper.shadow.css

```diff
@@ -4,7 +4,7 @@ SPopper {
   color: var(--intergalactic-text-primary, #191b23);
 }
 
-SPopper[keyboardFocused]:focus:after {
+SPopper[keyboardFocused][focusable]:focus:after {
```

Comment:
I am sure that it's better to display it by default and only hide in case of _condition_

Reply:
I don't understand, what is the difference between show it always and hide only if `keyboardFocused === false`

Reply:
added `hideFocus` prop